net-firewall/fwknop/fwknop-2.6.10.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Python extension supports only Python 2.
+# See https://github.com/mrash/fwknop/issues/167
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_OPTIONAL=1
+
+inherit autotools distutils-r1 eutils linux-info readme.gentoo-r1 systemd
+
+DESCRIPTION="Single Packet Authorization and Port Knocking application"
+HOMEPAGE="https://www.cipherdyne.org/fwknop/"
+SRC_URI="${HOMEPAGE}/download/${P}.tar.gz"
```

Comment:
> Please plaintext the beginning for easier copypasting.

Sorry, I don't understand what you mean by this.

Reply:
`SRC_URI="https://www.cipherdyne.org/fwknop/download/${P}.tar.gz"`

We try to avoid usage of unneeded variables here, for example they are banned in HOMEPAGE even though they work.

Reply:
Oh!  Sure.

I'll update for that and your other comments.  I will update the branch this PR is against, unless it would be better for me to make a fresh PR with only a single correct commit?

Reply:
You can squash your git commits and force-push in the current PR. Also you can use `Closes:` instead of `Bug:` to automatically close the linked bug while merging. 

I fixed those now, this is a good contribution. Thanks and welcome aboard!